promql/engine.go

```diff
@@ -2876,6 +2914,14 @@ func vectorElemBinop(op parser.ItemType, lhs, rhs float64, hlhs, hrhs *histogram
 				return 0, hlhs.Copy().Mul(rhs).Compact(0), true, nil
 			case parser.DIV:
 				return 0, hlhs.Copy().Div(rhs).Compact(0), true, nil
+			case parser.LTRIM:
+				trimmedHist := *hlhs
+				trimBuckets(&trimmedHist, hlhs, rhs, "LTRIM")
+				return 0, &trimmedHist, true, nil
```

Comment:
We should also consider how this behaves when the `bool` modifier is used.  
@beorn7, can you please clarify what the expected output should be in this case?

Reply:
I don't think the `</` and `>/` operators should be allowed with the `bool` modifier. It's a syntax error.

Reply:
Yeah, so we should disable it. Currently, it does not throw any syntax error.